roles/kubernetes-apps/policy_controller/meta/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+dependencies:
+  - role: download
+    file: "{{ downloads.calico_policy }}"
+    when: enable_network_policy and
+      kube_network_plugin in ['calico', 'canal']
+    tags: [download, canal, policy-controller]
+  - role: policy_controller/calico
+    when: kube_network_plugin == 'calico' and
```

Comment:
Yes, for Canal calico policy controller is a must, otherwise it's not Flannel, not really Canal :) So for Canal we always deploy policy controller ignoring enable_network_policy